src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -73,7 +73,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                        help='Default action to apply when no rule matches. Only applicable to Premium SKU.')
 
     with self.argument_context('acr import') as c:
-        c.argument('source', help="The source identifier in the format '[registry.azurecr.io/]repository[:tag]' or '[registry.azurecr.io/]repository@digest'.")
+        c.argument('source', help="The source identifier will be either a source image name or a fully qualified source.")
```

Comment:
I still would like us to highlight the fact that if `-r` specified, `--source` is just the image name; while `-r` is not specified, `--source` is the fully qualified image name, will defer the decision to @nathana1

Reply:
Ya